zipline/algorithm.py

```diff
@@ -2384,7 +2384,14 @@ def all_api_methods(cls):
 
 # Map from calendar name to default domain for that calendar.
 _DEFAULT_DOMAINS = {
-    'NYSE': domain.US_EQUITIES,
-    'TSX': domain.CA_EQUITIES,
-    'LSE': domain.GB_EQUITIES,
+    'XBRU': domain.BE_EQUITIES,
```

Comment:
If we have a list of built-in domains defined in the domain module, then this can probably turn into something like:

```python
_DEFAULT_DOMAINS = {d.calendar_name: d for d in domain.BUILT_IN_DOMAINS}
```
(We should make `calendar_name` public if we do ^).